homeassistant/components/flux_led/util.py

```diff
@@ -60,7 +60,7 @@ def _str_to_multi_color_effect(effect_str: str) -> MultiColorEffects:
         if effect.name.lower() == effect_str:
             return effect
     # unreachable due to schema validation
-    assert False  # pragma: no cover
+    raise RuntimeError  # pragma: no cover
```

Comment:
Can we remove `pragma no cover` here?
```suggestion
    raise AssertionError("Should be unreachable")
```

Reply:
Why? It has no use?

Reply:
I suggested it because it was in `.coveragerc`
If we keep `RuntimeError`, then maybe we can add it to `.coveragerc`

```
    # Don't complain if tests don't hit unreachable code
    raise RuntimeError("Should be unreachable")
```

Reply:
```suggestion
    raise RuntimeError("Should be unreachable")
```